extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -220,6 +228,19 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.analyticsGroup_ =
         this.instrumentation_.createAnalyticsGroup(this.element);
 
+    if (this.config_['transport'] && this.config_['transport']['iframe']) {
```

Comment:
Unless I'm missing something, you still need to ensure that only configs within the repo allow for iframe to declared, correct?